src/components/search/search.tsx

```diff
@@ -34,11 +40,27 @@ export interface SearchProps {
   nextMatchLabel?: string;
   prevMatchLabel?: string;
   searchResultsLabel?: string;
+  eventManager?: any;
+  upperBarManager?: UpperBarManager | undefined;
 }
 
+@withEventManager
 class SearchComponent extends Component<SearchProps> {
   private _inputField: InputField | null = null;
 
+  constructor(props: SearchProps) {
+    super(props);
+    this.props.eventManager?.listen(document, 'keydown', this.handleKeydownEvent);
+  }
+
+  private handleKeydownEvent = (event: KeyboardEvent) => {
+    if (event.keyCode === TAB && event.shiftKey && this._inputField?.base?.contains(document.activeElement)){
+      event.preventDefault();
+      //@ts-ignore
+      this.props.upperBarManager?.focusPluginButton('Transcript');
```

Comment:
why you use label of plugin button instead of ID (`this._transcriptIcon` in TranscriptPlugin class)?

Reply:
done